doc/z-chap03.xml

```diff
@@ -148,10 +178,33 @@
       <M>b &lt; b' &lt; a &lt; a'</M>.
     </Item>
   </List>
```

Comment:
Perhaps we might scrap the whole "equivalently" definition, since it's much more complicated and seems (to me) to add nothing.

Also, would it be good to use `a_1` and `a_2` instead of `a` and `a'`?  It's just that so much of the literature uses `i` for points on the top and `i'` for points on the bottom, it confused me for a bit here.

Both of these comments are suggestions, and since this PR only improves the situation, this won't stop me merging.